code-challenges.js

```diff
@@ -58,12 +141,67 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+
+//console.log(numberOfConnections.lastIndexOf(givenValue1))
+// output: 7
+//console.log(numberOfConnections.lastIndexOf(givenValue2))
+// output: 8
+
+//OR
+
+const location = (num1) => {
+    return numberOfConnections.lastIndexOf(num1)
+}
+
```

Comment:
Awesome! How could you refactor this to make it dynamic? (For example not need to hard code the `numberOfConnections` variable into the function?)